core/src/main/java/io/opencensus/stats/Measure.java

```diff
@@ -31,7 +35,10 @@
       Function<? super MeasureLong, T> p1);
 
   /**
-   * Name of measure, as a {@code String}.
+   * Name of measure, as a {@code String}. Should be a ASCII string with a length no greater than
+   * 255 characters.
+   *
+   * <p>Suggested format for name: {@code <web_host>/<path>}.
```

Comment:
Theses are some of our cross-language restrictions - we plan to put a document in place soon with more explanation. Might be worth adding a todo to add a link to the document when it becomes available.

There is still some discussion on this particular point, although details should be resolved soon.